test/snippet/core/detail/all_of_in_type_list.cpp

```diff
@@ -1,8 +1,8 @@
 #include <type_traits>
 
-#include <seqan3/core/detail/pack_algorithm.hpp>
-#include <seqan3/core/type_list/type_list.hpp>
 #include <seqan3/core/debug_stream.hpp>
+#include <seqan3/core/detail/pack_algorithm.hpp>
```

Comment:
Now I am confused. This snippet is called `all_of_in_type_list` but you are including `pack_algorithm`? Does pack_algorithm also include algorithms on type lists?

Reply:
pack algorithm is from a different header. So it was not part of this change.